app/config/roles.php

```diff
@@ -13,6 +13,8 @@
     'documents.write',
     'files.read',
     'files.write',
+    'buckets.read',
+    'buckets.write',
```

Comment:
note to myself what this is doing, because of missing collection but existing projects permissions.